python/pyproject.toml

```
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "okp4_grpc_client"
+version = "1.0.0"
```

The first version will be `1.0.0`, in order to make the release to work we need to specify a different version here.
```suggestion
version = "0.1.0"
```

---

Yes! Thx! :relaxed: